### ISSUE TITLE
"Made Village" Replacement

### ISSUE THREAD

**Issue by MirayXS** [kwsch/NHSE#186]
# "Made Village" Replacement
## Should change the "Made Village" option of misc category to "Resident Representative" to not get people confused

**Comment by kwsch** [comment]
It hasn't been confirmed that it actually makes the game recognize the flagged player as the Resident Representative. I'd rather not over-promise that it actually does that.

**Comment by MirayXS** [comment]
oh i tried before using that option, i unchecked it, loaded my game and in the passport i wasn't resident rep.. When i checked this after, i had the resident rep on my passport

**Comment by kwsch** [comment]
Right, but that doesn't mean the game will still treat the player as a Resident Representative. It might just be for display purposes on the passport.

**Comment by MirayXS** [comment]
that's a good think, i don't know then 😓

**Comment by MirayXS** [comment]
I’ll try to load my game with the “made villager” option unchecked, I’ll update you

**Comment by MirayXS** [comment]
Ok so it doesn’t remove you the resident rep from the game but from the passport
I was able to do all the resident representative things like building bridges
I also tried to delete save files trough the (-) button, and Tom nook said that I’am resident representative so I can’t delete my save data.

**Comment by kwsch** [comment]
Thanks for confirmation that it's just a cosmetic thing!

**Comment by MirayXS** [comment]
No problem! glad to help 🎉💯

**Comment by Hassiopeia** [comment]
Hey, sorry for replying to an old issue, but I just found that Player Flag 45 is responsible for setting the in-game Resident Representative. I'm sure there are other Player Flags that would also need switched if you're truly changing Representative from one player to another. I'm working on doing that now, but in the end I think I may just dump the Player Flags for my two villagers and then just swap them when I load them back in.

**Comment by vaaski** [comment]
I just switched the "Made Village" and flag 45 around for me and my wife, and it seemed to actually change the rep.
Now she can contribute stuff for building the shop, donate species for the museum, it says that she's the representative in her pass and she can't move to another island.
I no longer have the option to donate stuff for building the shop and it no longer says representative in my pass, and I am now able to move my player to another island.

I can't think of more ways to test this, but it seems as if flipping those two things pretty much changes who the representative is.

Oh one thing though - the progress for building the shop was reset, I had donated 30 hardwood, but my wife had to "start" the mission from the beginning again and was introduced to building the shop as if I had never started it.
